### PR TITLE
[Prometheus Remote Write Exporter for Cortex] Documentation Updates

### PR DESCRIPTION
**Pull request by ercl**
This PR updates the documentation in the package.

### CODE REVIEW

File: exporters/metric/cortex/README.md [hunk 1]
```diff
@@ -1,35 +1,121 @@
...
 ```go
+// Create a Config struct named `config`.
+
 pusher, err := cortex.InstallNewPipeline(config)
 if err != nil {
     return err
 }
 
-// Make instruments and record data
+// Make instruments and record data.
```
Comment (jmacd) [inline exporters/metric/cortex/README.md:49]: Maybe indicate that the SDK was installed as the global SDK, with:
```
// Make instruments and record data using `global.MeterProvider`.
```

**Review by jmacd — APPROVED** [review]
This is great!

File: exporters/metric/cortex/README.md (hunk 1 above)
Comment (ercl) [reply exporters/metric/cortex/README.md]: Done!

**Review by Aneurysm9 — APPROVED** [review]

**Review by MrAlias — APPROVED** [review]

**Comment by MrAlias** [comment]
@ercl can you sync with upstream again, this looks ready to merge.

**Pull request merged**